web/packages/teleterm/src/ui/DocumentAuthorizeWebSession/DocumentAuthorizeWebSession.story.tsx

```diff
@@ -33,65 +34,41 @@ import { DocumentAuthorizeWebSession } from './DocumentAuthorizeWebSession';
 
 export default {
   title: 'Teleterm/DocumentAuthorizeWebSession',
-};
-
-const doc: types.DocumentAuthorizeWebSession = {
-  uri: '/docs/e2hyt5',
-  rootClusterUri: rootClusterUri,
-  kind: 'doc.authorize_web_session',
-  title: 'Authorize Web Session',
-  webSessionRequest: {
-    redirectUri: '',
-    token: '',
-    id: '',
-    username: 'alice',
+  component: Story,
+  argTypes: {
+    isDeviceTrusted: { control: { type: 'boolean' } },
+    isRequestedUserLoggedIn: { control: { type: 'boolean' } },
+  },
+  args: {
+    isDeviceTrusted: true,
+    isRequestedUserLoggedIn: true,
   },
-};
+} satisfies Meta<StoryProps>;
```

Comment:
FWIW, I saw that in Storybook docs they prefer to define the default export as a separate variable with a specific type and only then do `export default meta`. It doesn't seem that `satisfies` is needed there nor was it added for this purpose.

https://storybook.js.org/docs/essentials/controls#choosing-the-control-type
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-9.html#the-satisfies-operator

Reply:
> It doesn't seem that it was for this purpose.

Oops 😅 I will have one more PR to device trust, I will correct it then.